lib/layout/dummyLayoutV2.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * dummyLayoutV2.js
+ *
+ * this creates a basic poorly laid out graph
+ *
+ * @author John O'Brien.
+ */
+
+
+// Create a graph instance.
+
+function buildGraph(graphs) {
```

Comment:
- Please document what the parameter of this method is
- I thought this was supposed to lay things out. Why is it refer to building. Am I missing something?